README.md

```diff
@@ -1 +1 @@
-Visit [https://primeflex.org](PrimeFlex) website for demos and documentation.
\ No newline at end of file
+Visit the [PrimeFlex website](https://primeflex.org) for demos and documentation.
```

Comment:
Duplicate with this maybe https://github.com/primefaces/primeflex/pull/181

Reply:
It is but I like @jepsar's change better so I can cancel mine. Can you link to this issue so it closes with the PR? https://github.com/primefaces/primeflex/issues/177

Reply:
> It is but I like @jepsar's change better so I can cancel mine. Can you link to this issue so it closes with the PR? #177

It's true, @jepsar can you link the issue to your PR adding in your first message `Fix #177` please?

Thanks both